tests/snmp/test_snmp_loopback.py

```diff
@@ -54,8 +54,12 @@ def test_snmp_loopback(duthosts, rand_one_dut_hostname, nbrhosts, tbinfo, localh
     for ip in config_facts[u'LOOPBACK_INTERFACE'][u'Loopback0']:
         loip = ip.split('/')[0]
         loip = ipaddress.ip_address(loip)
-        result = get_snmp_output(loip, duthost, nbr, creds)
-        if result is None: 
+        # TODO: Fix SNMP query over IPv6 and remove the below check.
+        if (isinstance(loip, ipaddress.IPv4Address)) == False:
```

Comment:
>( [](start = 11, length = 1)

remove the outside `()`, and use `not` instead of `== False`.
Hint: use linter to find this issue automatically, better integrated with an editor.

Reply:
Fixed as per comment.